src/vaadin-grid-data-provider-mixin.html

```diff
@@ -21,12 +25,19 @@
       this.pendingRequests = {};
     }
 
+    /**
+     * @return {boolean}
+     */
     isLoading() {
       return Object.keys(this.pendingRequests).length || Object.keys(this.itemCaches).filter(index => {
         return this.itemCaches[index].isLoading();
       })[0];
     }
 
+    /**
+     * @param {number} index
+     * @return {!GridItem}
```

Comment:
This may return `undefined` if there is no such row with given index (e.g. too big number)
```suggestion
     * @return {!GridItem | undefined}
```

Reply:
Done.